.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Could you remove this file?
Also, `package-lock.json`, too.
It's because the changes are not related to this PR.

Reply:
I have removed them